test/run.sh

```diff
@@ -104,6 +104,11 @@ setup_environment() {
     fi
   fi
 
+  # if the test is a benchmark we have to configure the environment
+  if [ "$cvmfs_benchmark" = "yes" ]; then
```

Comment:
That breaks if `cvmfs_benchmark` is not defined.  The trick to get around it is 

    if [ "x$cvmfs_benchmark" = "xyes" ]; then

(notice the additional x)